### PR TITLE
Add links to the community forum back

### PR DESCRIPTION
**Pull request by TomWFox**
Reverts parse-community/parse-community.github.io#37

### CODE REVIEW

**Comment by TomWFox** [comment]
I’ll hold off to merge until the forum is fully public.

**Comment by TomWFox** [comment]
I’ve just removed the link for server fault too as I checked the Parse tag and people aren’t getting answers there so it’s not super helpful to point them there.

File: _includes/header.html [hunk 1]
```diff
@@ -12,6 +12,9 @@ <h2>
         <a href="//docs.parseplatform.org/parse-server/guide/" target="_blank">
             <button class="large">Get Started</button>
         </a>
+        <a href="https://community.parseplatform.org" target="_blank">
```
Comment (acinader) [inline _includes/header.html]: remove the protocol, pretty sure that'll work right cross domain.

**Review by acinader — CHANGES_REQUESTED** [review]

File: _includes/header.html (hunk 1 above)
Comment (TomWFox) [reply _includes/header.html]: Right you are 👍

**Pull request merged**